guides/common/linkchecker.ini

```diff
@@ -18,6 +18,5 @@ ignore=example.com
   rhsso.com
   sources/
   atixservice.zendesk.com
-  # 6.14 docs are not yet published
-  access.redhat.com/documentation/en-us/red_hat_satellite/6.14
```

Comment:
seems this has broken linkchecker now, as 6.14 docs are not yet published

Reply:
I didn't quite think that through. The GA was postponed by one week, should I revert this part until then?